### PR TITLE
681 congress address handle

### PR DESCRIPTION
**Pull request by kfangw**
### Github Issue
#681 

### Background
CongressVoting(Result) Operations accepted only from CongressAddress

### Solution
- Any account can submit a transaction with CongressVoting(Result) operation
- The operation also charged transaction fee
- The account needed to be created before submitting the transaction.
- People knows what the Congress Address is.

However,
- To control the first several votings appropriately, only tx from the hard-coded account(Congress Address) is accepted temporary

### Possible Drawbacks
-

### CODE REVIEW

**Comment by kfangw** [comment]
@spikeekips @soonkuk @outsideris please review.

File: tests/client/congressvoting_test.go [hunk 1]
```diff
@@ -14,11 +14,14 @@ import (
...
 	"github.com/stellar/go/keypair"
 	"github.com/stretchr/testify/require"
+	"boscoin.io/sebak/lib/common"
 )
 
 func TestCongressVoting(t *testing.T) {
 
 	var (
+		CongressAddr  = "GAIJAH4FCEB3AWS2NVKRNDGVJ5QPD7VNOQNGZEPL2FAI6GPZQNKEXRWI"
+		CongressSecret = "SBUONPFGBX7BF76ZZ2NOWWCJPG6NQQOQS37IL27JCUBPFAXFRYS3VDI2"
```
Comment (soonkuk) [inline tests/client/congressvoting_test.go]: This `CongreeAddr` is used also in `constant.go`. If this address is used without change.
This secret key can be known to all.
How about change the `CongreeAddr` and `CongressSecret` of test to another keypair?

Comment (kfangw) [reply tests/client/congressvoting_test.go]: It is a client's testcode.
The Address should be the same as SEBAK's Congress Address.
When the SEBAK published, the address should be set by the environment variable.

Comment (kfangw) [reply tests/client/congressvoting_test.go]: I mean that The string set in the constant.go is just default code.
It should be changed when launching the network. @soonkuk

File: cmd/sebak/cmd/run.go [hunk 2]
```diff
@@ -488,6 +491,10 @@ func getTimeDuration(str string, defaultValue time.Duration, errMessage string)
 }
 
 func runNode() error {
+
+	// Set common variable
+	common.CongressAddr = flagCongressAddress
```
Comment (spikeekips) [inline cmd/sebak/cmd/run.go]: The other important shared variables are managed by `common.Config`.

File: lib/transaction/operation/congress_voting_result.go [hunk 3]
```diff
@@ -80,5 +80,5 @@ func (o CongressVotingResult) IsWellFormed(common.Config) (err error) {
 }
 
 func (o CongressVotingResult) HasFee() bool {
-	return false
+	return true
```
Comment (spikeekips) [inline lib/transaction/operation/congress_voting_result.go]: 👍

File: tests/client/congressvoting_test.go (hunk 1 above)
Comment (soonkuk) [reply tests/client/congressvoting_test.go]: @kfangw Ok I got it.

**Comment by kfangw** [comment]
@spikeekips @soonkuk  could you confirm this?

**Comment by kfangw** [comment]
@spikeekips PING~

File: lib/common/config.go [hunk 4]
```diff
@@ -50,5 +52,7 @@ func NewConfig(networkID []byte) Config {
 
 	p.HTTPCachePoolSize = HTTPCachePoolSize
 
+	p.CongressAddr = "GAIJAH4FCEB3AWS2NVKRNDGVJ5QPD7VNOQNGZEPL2FAI6GPZQNKEXRWI"
```
[the remark this answers is not in the document]
Comment (spikeekips) [reply lib/common/config.go]: @kfangw Please consider upper comment.

**Pull request merged**